tests/test_fsx.py

```diff
@@ -297,3 +297,156 @@ def test_fsx_create_backup_without_copy_tags(self):
         self.assertEqual(len(backups['Backups']), 1)
         expected_tags = [{'Key': 'test-tag', 'Value': 'backup-tag'}]
         self.assertEqual(expected_tags, backups['Backups'][0]['Tags'])
+
+
+class TestFSxBackup(BaseTest):
+    def test_fsx_backup_resource(self):
```

Comment:
this test really isn't asserting anything, thats not covered separately by other tests. to the extent that we have a empty policy resource query tests, ideally their targeting against particular augments specific to the resource. i think we can go ahead and remove this one.